Client/src/Pages/Uptime/Home/index.jsx

```diff
@@ -72,7 +72,7 @@ const UptimeMonitors = () => {
 				</Stack>
 				<Greeting type="uptime" />
 			</Box>
-			{noMonitors && <Fallback isAdmin={isAdmin} />}
+			{!loading && noMonitors && <Fallback isAdmin={isAdmin} />}
 			{loading ? (
 				<SkeletonLayout />
 			) : (
```

Comment:
We could turn these into a ternary:
Loading => Skeleton layout
Else => no monitors => Fallback
Else => has monitors

Reply:
I'm on board with that, do you want to make changes and push to my branch?

Reply:
Simplified and pushed